### PR TITLE
Nuevas Transducciones

### PR DESCRIPTION
**Pull request by KutonnoZer0**
Traducciones para:
- Disable Security Mitigations
- Enhanced Log
- Hot Plug USB
- Open Files

también se  agrego  "Nota:" en el archivo configedit y se cambio la palabra "Matar" por Terminar

### CODE REVIEW

**Comment by spencerjunraid** [comment]
There are conflicts with the main branch. I will look into it this weekend to see what the issue is.

**Comment by spencerjunraid** [comment]
Hola @KutonnoZer0 - OK. Encontré el conflicto. Puedes ver esto: https://github.com/unraid/lang-es_ES/pull/6/conflicts ?

Existen diferencias en el número de líneas entre el original y su archivo. ¿Puedes volver a enviar el PR con las mismas cantidades de línea?

**Comment by KutonnoZer0** [comment]
Si puedo re enviar el PR de nuevo, pero creo que el conflicto esta en 

<<<<<<< master
note editing a binary file could have unintended consequences=**Nota**: Editar un archivo binario podría tener consecuencias no deseadas
=======
note editing a binary file could have unintended consequences=**nota:** editar un archivo binario podría tener consecuencias no deseadas

**Comment by spencerjunraid** [comment]
hmm. @bergware - Could you take a look at what the conflict is here and let me know how to resolve it?

**Comment by bergware** [comment]
The line "note editing a binary file could have unintended consequences=" is translated twice (different translations)

note editing a binary file could have unintended consequences=Nota: Editar un archivo binario podría tener consecuencias no deseadas
=======
note editing a binary file could have unintended consequences=nota: editar un archivo binario podría tener consecuencias no deseadas

You need to edit the file and keep the translation which is correct and remove the other one (see **Resolve conflicts**)

**Comment by spencerjunraid** [comment]
OK- thanks @bergware. I will get a moderator from the forums to check the translations and then merge upon review.

File: Disable Security Mitigations/disablesecurity.txt [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-Mitigations currently enabled=
-Status=
-will be updated after rebooting the server=
-Enable Mitigations=
-Disable Mitigations=
-Disable Security Mitigations A Reboot Is Required For Changes To Take Effect=
-A Reboot Is Required For Changes To Take Effect=
-Mitigations will be enabled after a reboot=
-Mitigations will be disabled after a reboot=
+Disabling Spectre, Meltdown and Zombieload MDS mitigations may improve your CPU speed But these are security risks for certain workloads, and disabling the mitigations is purely all at your own risk=Deshabilitar las mitigaciones de Spectre, Meltdown y Zombieload MDS puede mejorar la velocidad del CPU, estos son riesgos de seguridad para ciertas cargas de trabajo; La desactivación de las mitigaciones es bajo su propio riesgo
```
Comment (sonoseco) [inline Disable Security Mitigations/disablesecurity.txt:1]: Would make the following changes: `Deshabilitar las mitigaciones de Spectre, Meltdown y Zombieload MDS puede mejorar la velocidad del CPU, pero hay riesgos de seguridad para ciertas cargas de trabajo; La desactivación de las mitigaciones es bajo su propio riesgo`

Comment (KutonnoZer0) [reply Disable Security Mitigations/disablesecurity.txt]: Got it!

**Comment by spencerjunraid** [comment]
Thank you very much @sonoseco. @KutonnoZer0 - if you could update these suggestions and resubmit, I'll merge. Thanks!

**Comment by KutonnoZer0** [comment]
> * Change the translation for the word "Match", i.e. `Custom Match=Coincidencia Personalizada` on line 4, I would translate to: `Custom Match=Pareo Personalizado`

This one is hard to translate because the work match is translate as a Igual, partido or Encuentro.
Partido is use for like Game Match, the other one Igual didn't sound good like, Igual personalizado, but I guess I could change it for Encuentro personalizado if that sound good to you.

> * Make sure to follow capitalization, i.e. `Color Selection Chart=Carta de selección de Colores` should be `Color Selection Chart=Carta de Selección de Colores `. There are many examples like this so most if those lines should be updated.

For this one I could change it for "Palestra de Selección de Colores"

**Comment by serfriz** [comment]
Hello, I haven't been able to go over everything, but from reading the comments I think "Coincidencia" should be left as is. Also, I saw some translations that are not following the style used for the main OS translations (i.e. array was kept as array instead of matriz). And it is important to make sure that the meaning of the translated phrases are kept as close as possible, ie: Kill Process=/=Proceso de Terminación. Kill Process should be Finalizar Proceso or Terminar Proceso.

**Comment by spencerjunraid** [comment]
I won't merge anything until you have time @serfriz.

**Comment by sonoseco** [comment]
@serfriz `Coincidencia` does not add up to me, I don’t understand Custom Match as Coincidencia Personalizada, unless there is something very specific to this content where using a different word meaning is needed. But I’ll leave this up to others to decide.

I hope to have more time to review other files later today.

Thanks.

**Comment by serfriz** [comment]
@sonoseco I just installed the plugin to make sure, and that field refers to words/phrases that if found in the syslog file are highlighted. So the script looks through the entire log for phrases that match (que coincidan con) the user defined ones. I don't know how to explain it better, but for me a pareo is this: https://es.wikipedia.org/wiki/Pareo.

imagen

Actually in this case I would use the plural since it allows for multiple strings. Custom Match = Coincidencias Personalizadas. And for Custom syslog match strings = Expresiones personalizadas que se encuentren en el registro del sistema o Expresiones personalizadas presentes en el registro del sistema.

**Comment by sonoseco** [comment]
@serfriz Within that context it makes more sense and I agree with your comments. Thank you for putting it all together, I’ve been unable to install the beta/plugins but will give it a shot this weekend and comment further once I have more “hands-on”.

**Comment by spencerjunraid** [comment]
Checking back in... should I merge and then we can add changes after or I can close and we can resubmit a PR with changes?

**Comment by spencerjunraid** [comment]
Merging.

**Pull request merged**